bundler/man/bundle-config.ronn

```diff
@@ -333,21 +333,21 @@ Bundler supports overriding gem sources with mirrors. This allows you to
 configure rubygems.org as the gem source in your Gemfile while still using your
 mirror to fetch gems.
 
-    bundle config set mirror.SOURCE_URL MIRROR_URL
+    bundle config set --local mirror.SOURCE_URL MIRROR_URL
```

Comment:
Not sure if better to show `--local` or `--global` for mirror and authentication (just below).
But at least it seems better to show one of them, so users can consider whether they want it local or global and not rely on the default which might not do the right thing.

Reply:
Agreed. Since we are unsure, let's go with `--global` I guess to match current "flagless" behaviour?

Reply:
OK, switched to `--global` for mirror & auth config commands.